README.md

```diff
@@ -61,7 +61,7 @@ find and fix the errors.
 |-------------|------------------|--------------------|----------------|
 | `isort`     | `pyproject.toml` | `isort . --check`  | `isort .`      |
 | `black`     | `pyproject.toml` | `black . --check`  | `black .`      |
-| `flake8`    | `.flake8`        | `flake8`           | none           |
+| `flake8`    | `.flake8`        | `flake8 ./**/*.py` | none           |
```

Comment:
![obraz](https://user-images.githubusercontent.com/34098778/166573418-da9f3432-4f26-4eee-b32d-0372923409ce.png)

Reply:
using `flake8` only checks every single file in the project which is not required and takes a lot of time. @jplewa

Reply:
Okay, makes sense